aten/src/ATen/native/ScatterGather.cpp

```diff
@@ -0,0 +1,85 @@
+#include <tuple>
+#include <algorithm>
+#include <ATen/ATen.h>
+#include <c10/core/WrapDimMinimal.h>
+
+namespace at { namespace native {
```

Comment:
This code got moved, any substantive changes?

Reply:
Do you mean the `gather` and `gather_out`? Currently is nothing but a simple renaming, in the next broadcasting will be added here.